app/models/admin_dashboard_data.rb

```diff
@@ -211,6 +211,15 @@ def github_config_check
     end
   end
 
+  def pwa_config_check
+    unless SiteSetting.large_icon.present? && SiteSetting.large_icon.width == 512 && SiteSetting.large_icon.height == 512
+      return I18n.t('dashboard.pwa_config_icon_warning', base_path: Discourse.base_path)
+    end
+    unless SiteSetting.short_title.present? && SiteSetting.short_title.size <= 12
```

Comment:
Should we force a max length for that setting instead of adding a warning? `site_settings.yml` has an option to define the `max` length.

Reply:
Good Idea!

Added a max to the setting, but letting the warning there for now so we can guide sites who already filled it.